.travis.yml

```diff
@@ -15,6 +15,9 @@ services:
 
 addons:
   postgresql: '9.6'
+  apt:
+    packages:
+    - postgresql-9.6-postgis-2.3
```

Comment:
Do we still need it in Travis?

Reply:
I found it at https://docs.travis-ci.com/user/database-setup/#using-postgis

Reply:
Ok, but do you really need to do this in order to run tests?

Reply:
I checked it out - https://travis-ci.org/FedorSelitsky/eventrack/builds/440329100 and I really need it.